root.sh

```diff
@@ -1,7 +1,7 @@
 package: ROOT
 version: "%(tag_basename)s"
-tag: "v6-30-01-alice5"
-source: https://github.com/alisw/root.git
+tag: "v6-32-00"
+source: https://github.com/root-project/root.git
```

Comment:
Note that we have custom commits in our ALICE specific branches that would be skipped with this move. 

Also, we need to make sure that any bump in ROOT contains the fix for this issue: https://github.com/root-project/root/issues/15579.

Reply:
ok, then I guess that we need to get 6.32.00 inside `alisw` and apply the patches. @ktf ?

Reply:
I have forked alice/v6-32-00 and added the remaining patches from v6-30-01-alice5. @sawenzel can you open a PR to the alisw/root branch with the extra commits you need?